spec/refile/postgres/backend_spec.rb

```diff
@@ -64,6 +64,18 @@
     end
   end
 
+  describe "Orphaned large object cleaning" do
```

Comment:
a test showing that other unreferenced large objects are vacuumed would be handy. but nbd.